source/images/mozilla-block-white.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 2000 571.9"><defs><style>.cls-1{fill:#fff}</style></defs><path class="cls-1" d="M746.2 254.1c-36.6 0-59.4 27.2-59.4 74.3 0 43.3 20 76.6 58.8 76.6 37.2 0 61.6-30 61.6-77.7 0-50.5-27.2-73.2-61-73.2zM1727.2 380c0 16.1 7.8 28.9 29.4 28.9 25.5 0 52.7-18.3 54.4-59.9-11.6-1.7-24.4-3.3-36.1-3.3-25.5-.1-47.7 7.1-47.7 34.3z"/><path class="cls-1" d="M0 0v571.9h2000V0zm581.9 454.4H477V313.5c0-43.3-14.4-59.9-42.7-59.9-34.4 0-48.3 24.4-48.3 59.4v87h33.3v54.4H314.5V313.5c0-43.3-14.4-59.9-42.7-59.9-34.4 0-48.3 24.4-48.3 59.4v87h47.7v54.4H118.7V400H152V258.5h-33.3v-54.4h104.8v37.7c15-26.6 41.1-42.7 76-42.7 36.1 0 69.3 17.2 81.6 53.8 13.9-33.3 42.2-53.8 81.6-53.8 44.9 0 86 27.2 86 86.5V400H582v54.4zm161.5 5.5c-77.1 0-130.4-47.1-130.4-127 0-73.2 44.4-133.7 134.3-133.7S881 259.6 881 329.5c0 79.9-57.7 130.4-137.6 130.4zm403.5-5.5H928.3l-7.2-37.7 137.6-158.1h-78.2l-11.1 38.8-51.6-5.6 8.9-87.7h219.7l5.6 37.7L1013.2 400h81l11.7-38.8 56.6 5.5zm142.8 0h-74.9v-89.9h74.9zm0-160.4h-74.9v-89.9h74.9zm46.8 160.4l108.2-381.7h70.5L1407 454.4zm145.4 0L1590 72.7h70.5l-108.2 381.7zm384 5.5c-33.3 0-51.6-19.4-54.9-49.9-14.4 25.5-39.9 49.9-80.4 49.9-36.1 0-77.1-19.4-77.1-71.6 0-61.6 59.4-76 116.5-76a341.08 341.08 0 0 1 41.1 2.2v-8.3c0-25.5-.6-56-41.1-56-15 0-26.6 1.1-38.3 7.2l-8.1 28.2-57.1-6.1 9.8-57.6c43.8-17.8 66-22.7 107.1-22.7 53.8 0 99.3 27.7 99.3 84.9v108.7c0 14.4 5.6 19.4 17.2 19.4a37.6 37.6 0 0 0 10.5-1.7l.6 37.7a97 97 0 0 1-45.1 11.7z"/></svg>
```

Comment:
looks like this still wants an svgo pass? the svg id, data-layer, defs etc should all be removable/compactable

Reply:
Odd. I thought I did. Will give it another pass.

Reply:
Ah I did already run `svgo` with this. Will try running it with `--multipass --pretty --enable=removeTitle` options passed in.

Reply:
in that case let's leave this as is, and look at SVGO in a separate cleanup issue =)

Reply:
@Pomax I think we need `id` and `defs` for this to apply the white fill. I couldn't find any [plugin](https://github.com/svg/svgo#what-it-can-do) to remove `data-name="Layer 1"` so I've removed manually myself.

Reply:
Ticket filed: https://github.com/mozilla/foundation.mozilla.org/issues/4979